src/Pos/Txp/Worker.hs

```diff
@@ -77,6 +79,8 @@ queryTxsWorker = worker queryTxsSpec $ \sendActions -> do
             action `catch` handler
     action `catch` handler
 
+type TxIdT = Tagged TxMsgContents TxId
```

Comment:
Because we need Specific `Message` instance for `InvMsg key` and `TxId` is just `type` over `AsbtractHash`